.gitignore

```diff
@@ -3,6 +3,8 @@
 .vs/
 .vscode/
 build/
+.idea/
+.DS_Store
```

Comment:
Could you move these 2 just above the `build/`.

Reply:
Sure will do that.

Reply:
Done!